s5_subwords/run.sh

```diff
@@ -278,16 +308,26 @@ if [ $stage -le 8 ] && $decode_gmm; then
   echo "          Decoding tri3                          "
   echo ============================================================================
   tri=tri3b
-  utils/mkgraph.sh data/$lang/lang_test \
-                   exp/$lang/$tri \
-                   exp/$lang/$tri/graph || exit 1;
-
+  
+  if [ $lm_tool == 'srilm' ]; then
+    utils/mkgraph.sh data/$lang/lang_test \
+                     exp/$lang/$tri \
+                     exp/$lang/$tri/graph_${lm_tool} || exit 1;
+
+  elif [ $lm_tool == 'kenlm' ]; then
+    utils/mkgraph.sh data/$lang/local/lang_${lm_order}g \
+                     exp/$lang/$tri \
+                     exp/$lang/$tri/graph_${lm_tool} || exit 1;
+  fi
+  
+  # Should I add this to decode??
+  #  --config conf/decode.config
```

Comment:
Yes, if you want to set specific values. I think I've usually used what's been in the recipe I'm working with. So if it is included there, then I include it.